packages/contracts/contracts/engine/V3/GenArt721CoreV3_Engine.sol

```diff
@@ -421,7 +421,10 @@ contract GenArt721CoreV3_Engine is
             engineConfiguration.tokenName,
             engineConfiguration.tokenSymbol
         );
-
+        // update minter if populated
+        if (engineConfiguration.minterFilterAddress != address(0)) {
+            minterContract = engineConfiguration.minterFilterAddress;
```

Comment:
@ryley-o Note - I think we need to emit the `MinterUpdated` event here but we ran out of space 💀 C

Reply:
Resolved offline 🙏